pkg/features/versioned_kube_features.go

```diff
@@ -496,6 +496,7 @@ var defaultVersionedKubernetesFeatureGates = map[featuregate.Feature]featuregate
 	MemoryManager: {
 		{Version: version.MustParse("1.21"), Default: false, PreRelease: featuregate.Alpha},
 		{Version: version.MustParse("1.22"), Default: true, PreRelease: featuregate.Beta},
+		{Version: version.MustParse("1.32"), Default: true, PreRelease: featuregate.GA},
```

Comment:
I think it should be 
```
{Version: version.MustParse("1.32"), Default: true, PreRelease: featuregate.GA, LockToDefault: true},
```

But I'll doublecheck because of https://github.com/kubernetes/enhancements/blob/master/keps/sig-architecture/4330-compatibility-versions/README.md

/cc @SergeyKanzhelev

Reply:
Yes, you're right. I also had to run  `hack/update-featuregates.sh` since it uses it. checkout the commits